CriticalMass/SettingsSection.swift

```diff
@@ -8,126 +8,204 @@
 
 import UIKit
 
-enum Section: CaseIterable {
-    typealias AllCases = [Section]
+/** TODO: Refactor to a more versataile API like:
+Section(title: "NAME", rows: [SwitchRow(title:subTitle:actionClosure:), OptionRow]).....
```

Comment:
I think this will be done in a separate task, right?

Reply:
Yes. Just wanted to remind me of it.